components/generic/Select/Select.tsx

```diff
@@ -9,25 +8,23 @@ type Option = {
 type SelectProps = {
   name: string;
   label: string;
-  value: string;
   options: Option[];
-  onChange: ChangeEventHandler<HTMLSelectElement>;
 };
 
-export const Select = ({ name, label, value, options, onChange }: SelectProps) => {
+export const Select = ({ name, label, options }: SelectProps) => {
```

Comment:
Those props will be used and they are necessary to make this component generic, add them.